README.md

```diff
@@ -1,26 +1,55 @@
-# README
 
-This README documents steps necessary to get the
-application up and running.
 
-* Clone the repo locally
+# Isilon Tracker
 
-* cd into cloned repo and bundle install, updating ruby if needed
+This application creates a filetree view of assets stored in the Isilon file storage system. It allows for updating of 
+metadata for each object ingested from CSV files exported from the Isilon server. Metadata is meant to facilitate the 
+transfer of all data from the Isilon to a new storage system, giving users the ability to track migration status, assign 
+users to assets, and assign assets to digital collections for organization and tracking.
+
+## Getting Started
+
+### Install the application
+
+```bash
+git clone git@github.com:tulibraries/isilon-tracker
+cd isilon-tracker
+bundle install
+```
+
+* Initialize the db and populate wtih seeds from config
+
+```bash
+bundle exec rails db:setup
+```
 
-* Run rails db:setup to initialize db and populate wtih seeds from config
+* Ingest sample sample data from the repo, if needed (file included in the repo)
 
-* Run rails sync:assets["scan_output.applications-backup.csv"] (included in the repo) for sample data
+```bash
+rails sync:assets["scan_output.applications-backup.csv"]
+```
 
-* Run bin/dev to start the development server and watch for css/js changes
+(note: in zsh shells, it may be necesary to escape the brackets in the above command. You can aavoid this by setting 'setopt nonomatch' in your .zshrc file)
```

Comment:
```suggestion
(note: in zsh shells, it may be necesary to escape the brackets in the above command. You can avoid this by setting 'setopt nonomatch' in your .zshrc file)
```